tee-worker/core-primitives/node-api/api-client-types/src/lib.rs

```diff
@@ -42,6 +45,31 @@ pub type ParentchainExtrinsicParamsBuilder = PlainTipExtrinsicParamsBuilder;
 pub type ParentchainUncheckedExtrinsic<Call> =
 	UncheckedExtrinsicV4<Call, SubstrateDefaultSignedExtra<PlainTip>>;
 
+#[derive(Clone, Eq, PartialEq, Debug)]
+pub struct ParentchainUncheckedExtrinsicWithStatus<Call: Decode> {
+	pub xt: UncheckedExtrinsicV4<Call, SubstrateDefaultSignedExtra<PlainTip>>,
+	pub status: bool,
+}
```

Comment:
here we defined `ParentchainUncheckedExtrinsic<Call>`, use it.

Reply:
i move it `itp_types`. Maybe it is more reasonable